assembl/static2/js/app/components/common/participateButton.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import { Translate } from 'react-redux-i18n';
+import { getIsDebateStarted } from '../../utils/timeline';
+
+type participateButtonProps = {
+  displayPhase: Function,
+  timeline: Timeline,
+  btnClass: string
+};
+
+export const participateButton = ({ displayPhase, timeline, btnClass }: participateButtonProps) => (
+  <div>
```

Comment:
Note that you can use `React.Fragment` if you don't want this `div`

Reply:
Please capitalize the component name `ParticipateButton`. You should just name the flow type `Props` when you have only one component in a file. This is what we do in other files.